pure_changelog.md

```diff
@@ -46,6 +46,7 @@
 * Fixed redundant modem polling for call states
 * Fixed selecting SIM during onboarding
 * Fixed broken text on SIM selection screen during onboarding
+* Fixed broken text on the PIN input screen
```

Comment:
I think that in the future, we could write which languages were affected. Just a minor thing, not bothering you to change it now :)

Reply:
done